.github/workflows/functions.yml

```
@@ -236,4 +236,4 @@ jobs:
     - name: PodLibLint Functions Cron
       run: |
         scripts/third_party/travis/retry.sh scripts/pod_lib_lint.rb \
-          FirebaseFunctions.podspec --platforms=${{ matrix.target }} --use-static-frameworks
+          FirebaseFunctions.podspec --platforms=${{ matrix.target }} --use-static-frameworks
```

nit: undo change to this file

```suggestion
          FirebaseFunctions.podspec --platforms=${{ matrix.target }} --use-static-frameworks

```